src/parser/rogue/outlaw/modules/features/RollTheBonesCastTracker.js

```diff
@@ -79,13 +71,12 @@ class RollTheBonesCastTracker extends Analyzer {
     if(!event || !event.classResources){
       return;
     }
-    const cpCost = getResource(event.classResources, RESOURCE_TYPES.COMBO_POINTS.id).cost;
     const refresh = this.lastCast ? event.timestamp < (this.lastCast.timestamp + this.lastCast.duration) : false;
 
     // All of the events for adding/removing buffs occur at the same timestamp as the cast, so this.selectedCombatant.hasBuff isn't quite accurate
     const appliedBuffs = ROLL_THE_BONES_BUFFS.filter(b => this.energyCapTracker.combatantHasBuffActive(b.id));
 
-    let duration = ROLL_THE_BONES_BASE_DURATION + (ROLL_THE_BONES_CP_DURATION * cpCost);
+    let duration = 30000;
```

Comment:
Is this duration always 30 seconds regardless of how many combo points you use?

Also should avoid using magic numbers. I would assign this to a constant like it was before.

Reply:
Yes. Roll the bones is a flat 30 seconds now.

Reply:
Then put that in a constant instead of using the number directly